include/swift/AST/SILOptimizerTypeIDZone.def

```diff
@@ -20,3 +20,6 @@ SWIFT_REQUEST(SILOptimizer, ExecuteSILPipelineRequest,
 SWIFT_REQUEST(SILOptimizer, LoweredSILRequest,
               std::unique_ptr<SILModule>(ASTLoweringDescriptor),
               Uncached, NoLocationInfo)
+SWIFT_REQUEST(SILOptimizer, ComputeFieldIndicesRequest,
```

Comment:
Hmm, ideally we'd have a SIL request evaluator zone for this. I think for now at least it might be better off placed in the `TypeChecker` zone, as it shouldn't depend on anything outside of Sema.

Reply:
ok. I can move it to the type checker